### DISCUSSION TITLE
Team의 팀원 관련 코드를 MemberTeam으로 분리한다.

### DISCUSSION THREAD

**Discussion by lcqff** [BDD-CLUB/01-doo-re-back#182] [논의 사항]
저희 폴더 구조는 도메인형 패키지 구조를 따라 도메인별로 코드를 분리해 사용하고 있습니다. 
Team 도메인과 MemberTeam(팀원) 도메인의 구분이 명확하지 않은 것 같습니다 (Study와 Participant(스터디원)의 구분이 명확한거에 비해)
MemberTeam 과 관련된 코드 (팀원 가입 등등)을 MemberTeam으로 옮기는게 좋아보입니다.

**Comment by JJimini** [comment]
팜님이 생각하는 **MemberTeam과 관련된 코드** 라는 것이 정확히 어떤 코드들을 의미하는 것인지 궁금합니다! 해당 discussion을 생각해보기 위해선 이 부분을 명확히 인지한 후 생각해보아야 할 것 같습니다.

**Reply by lcqff** [reply]
팀원과 관련된 CRUD 입니다! 팀원 추가, 삭제, 조회, 수정 등등등...

**Reply by JJimini** [reply]
좋습니다!
